.github/workflows/release-automation.yml

```diff
@@ -0,0 +1,41 @@
+name: Update flyteidl version
```

Comment:
Do we want to take this as part of another PR or is this required for your current one.

Reply:
no it's small one. Just a copy paste workflow